bin/quickpkg

```diff
@@ -43,6 +46,19 @@ def quickpkg_atom(options, infos, arg, eout):
 	fix_metadata_keys = ["PF", "CATEGORY"]
 
 	try:
+		binpkg_compression = settings.get("BINPKG_COMPRESSION", "bzip2")
+		compression = _compressors[binpkg_compression]
+	except KeyError as e:
+		eout.eerror("Unsupported compression method: %s" % e.args[0])
+		del e
+		return
+
+	if subprocess.call(["type", compression["decompress"]], shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE) != 0:
+		missing_package = compression["package"]
+		eout.eerror("File compression unsupported %s. Missing package: %s" % (binpkg_compression, missing_package))
+		return
```

Comment:
I can probably leave it out of quickpkg, as binarytree.__init__ will raise an exception.